src/libraries/Microsoft.PowerFx.Core/IR/IRTranslator.cs

```diff
@@ -389,6 +392,16 @@ private static IntermediateNode ReplaceBlankWithZero(IntermediateNode arg)
                 return convertedNode;
             }
 
+            /// <summary>
+            /// Wraps node arg => Truc(Coalesce(arg , 0)).
+            /// </summary>
+            private static IntermediateNode TruncatePreProcessor(IntermediateNode arg)
+            {
+                var blankToZeroNode = ReplaceBlankWithZero(arg);
+                var truncateNode = new CallNode(blankToZeroNode.IRContext, BuiltinFunctionsCore.Trunc, blankToZeroNode);
```

Comment:
>Trunc [](http://example.com/codeflow?start=96&length=5)

fyi - double checked that Sql does indeed include this one, so we should be good.

Reply:
I checked, it is there.